game/src/main/scala/io/aigar/model/TeamRepository.scala

```diff
@@ -25,6 +26,10 @@ class TeamRepository(databaseName: Option[String]) {
     TeamDAO.findTeamBySecret(db, teamSecret)
   }
 
+  def updateScore(team_id: Int, value: Int): Unit ={
```

Comment:
Similar, I think we should be explicit about the fact that this *adds* to the score, in retrospect.